src/main/scala/io/shiftleft/fuzzyc2cpg/SourceFiles.scala

```diff
@@ -0,0 +1,27 @@
+package io.shiftleft.fuzzyc2cpg
+
+import better.files._
+
+object SourceFiles {
+
+  /**
+    * For a given array of input paths, determine all C/C++
+    * source files by inspecting filename extensions.
+    * */
+  def determine(inputPaths : List[String])  : List[String]= {
+
+    def hasSourceFileExtension(file : File) : Boolean = {
+      val ext = file.extension
+      ext.contains(".c") || ext.contains(".cpp") || ext.contains(".h") || ext.contains(".hpp")
```

Comment:
Objective C/C++ files have `.m` and `.mm` extensions. I think we should search for them also

Reply:
Which makes me think, do we support that? (https://en.wikipedia.org/wiki/Objective-C#Interface)

Reply:
I can add .m and .mm, but I've never tried parsing objective c with this parser. Let me do that in a separate PR though.